rules/file-contents-config.json

```diff
@@ -11,6 +11,11 @@
       "type": "array",
       "items": { "type": "string" }
     },
+    "branches": {
+      "type": "array",
+      "items": { "type": "string" },
+      "default": ["default"]
```

Comment:
`default` is in this case not the actual branch-name but a magic label, the default of the repository.

Did I understand that correctly?

Reply:
Indeed!